test/test_torrent.cpp

```diff
@@ -399,8 +399,8 @@ TORRENT_TEST(queue)
 	pack.set_bool(settings_pack::disable_hash_checks, true);
 	lt::session ses(pack);
 
-	torrent_handle torrents[5];
-	for(int i = 0; i < 5; i++)
+	torrent_handle torrents[6];
```

Comment:
I think it would be nicer to make this a ``std::vector<torrent_handle> torrents;``. and then use ``torrents.push_back()`` when adding a new torrent, rather than directly indexing into it